mmcv/utils/progressbar.py

```diff
@@ -172,3 +172,35 @@ def track_parallel_progress(func,
     pool.close()
     pool.join()
     return results
+
+
+def track_iter_progress(tasks, bar_width=50, **kwargs):
+    """Track the progress of tasks execution with a progress bar.
+
+    Tasks are done with a simple for-loop.
+
+    Args:
+        func (callable): The function to be applied to each task.
```

Comment:
Incorrect argument.

Reply:
Sorry for missing these, I have updated a new version of the docs and code comments.